src/App.tsx

```diff
@@ -10,23 +10,46 @@ import Leaderboard from "./pages/Leaderboard/Leaderboard";
 import Profile from "./pages/Profile/Profile";
 import Register from "./pages/Register/Register";
 import Account from "./pages/Account/Account";
+import { getDocs, query, collection, getFirestore } from "firebase/firestore";
+import { app } from "./firebase";
+import { UserProfile } from "./mockData/mockTribe";
+import { useState, useEffect } from "react";
 
 const App = () => {
+  const [fetchedTribe, setFetchedTribe] = useState<UserProfile[]>([]);
+
+  useEffect(() => {
+    const fetchUsers = async () => {
+      const db = getFirestore(app);
```

Comment:
The db initialisation can be moved to the `firebase.ts` and exported. The reason for this is we do not want to initialise it every time we use it, multiple pages will use the db so it makes sense to create it once.

Have a look at [setup-file-upload](https://github.com/nology-tech/wov-client-project/blob/setup-file-upload/src/firebase.ts) to see who it is don with the storage.

Reply:
Hi Charlie, we've moved the db initialization to firebase.ts as requested.